main.go

```diff
@@ -321,6 +321,6 @@ func renderSchedulerManifests(schedManifests schedmanifests.Manifests, imageSpec
 	klog.InfoS("Updating scheduler manifests")
 	mf := schedManifests.Clone()
 	schedstate.UpdateDeploymentImageSettings(mf.Deployment, imageSpec)
-	schedstate.UpdateDeploymentConfigMapSettings(mf.Deployment, schedManifests.ConfigMap.Name)
+	schedstate.UpdateDeploymentConfigMapSettings(mf.Deployment, schedManifests.ConfigMap.Name, schedstate.GetConfigMapHash(schedManifests.ConfigMap))
```

Comment:
this is one of the cases on which we can use a temporary `cmHash` like you do in `numaresourcesscheduler_controller.go`

Reply:
This is exactly the same to what I'm doing in ```numaresourcesscheduler_controller.go``` - using compute the hash from the manifests instead of the object from the API server.